src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/ModifiedControlVariableCheckTest.java

```diff
@@ -131,6 +131,37 @@ public void testImproperToken() {
         }
     }
 
+    @Test
+    public void testMutation() throws Exception {
```

Comment:
Can we name this test something more meaningful. Your input has various tokens so it could be called `testVariousTokens` or such.

Reply:
done